opendata-mvcr/ontoGrapher
# Refactor elements and terms

## Info:
* This PR is really big, sorry about that. The refactoring changes the way elements (= terms) are recorded, which is a big change to the codebase.
* Closes #375 
* Closes #371 
* Closes #372 
* Closes #366 
* Closes [#AL-62](https://github.com/opendata-mvcr/sgov-assembly-line/issues/62).
* WorkspaceElements now have IRIs instead of UUIDs.
* Removed unused functions.
* Removed the Folder datatype.
* Improved overall stability of loading and saving diagrams.
* All diagrams are now loaded during workspace reconstruction.
* Removed unsuable searching in creation modals - it will be reimplemented in a later PR.

## Testing Guide
* #375 - Perform changes of cardinalities and see that they do (i) not crash the app, (ii) persist on refreshes.
* #371 - Create instrinsic tropes both in Compact and Full view and see that they do (i) not crash the app, (ii) persist on refreshes.
* #372 - Same as #375. 
* #366 - Create terms and see that they are visible in the left panel in Compact view desping having no type stereotypes.
* AL-62 - Change path of new & existing relationships and see that the paths persist on refreshes.
